### ISSUE TITLE
Possibly a file size issues

### ISSUE THREAD

**Issue by ajaycj1** [PlasmaFAIR/TDotDat#11]
When I try to upload the nonlinear field file (same one in Google drive), the page times out.
The file size is 2.6 GB.
Last week I used to get a 'file size too large' error.

**Comment by ZedThree** [comment]
Currently the upload limit is set to 100 MB. Not sure why it's not giving an error message.

Can you try with the latest version of the upload script? If you don't specify the output files, it should still be able to extract and upload the fluxes

**Comment by ajaycj1** [comment]
When I only specify the input file and don't specify any output files, the upload is successful but it doesn't look like any output files were uploaded and fluxes extracted.

When I try to upload the field file, the exact error I see, which I called the time out error,
 is attached below.

![Screenshot at 2022-10-31 17-25-01](https://user-images.githubusercontent.com/109351618/199070825-32e205b6-0642-41d3-9046-1de091462ef1.png)

**Comment by ZedThree** [comment]
Yes, the output files won't be uploaded in this case -- this will be a limitation until we secure sufficient storage.

The fluxes should've been extracted though, I'll look into it
